Developed in-house, not fed back.
Solaris specific - mysql.sock is located in different path than
it is in Linux.

--- a/doc/config/conf.d/mysql_vhost.conf
+++ b/doc/config/conf.d/mysql_vhost.conf
@@ -12,7 +12,7 @@ ## Either set the the socket or host (an
 ##
 ## Local path to the mysql socket
 ##
-#mysql-vhost.sock           = "/var/lib/mysql/mysql.sock"
+#mysql-vhost.sock           = "/tmp/mysql.sock"
 
 ##
 ## Host of the MySQL server. 
